cli/README.md

```diff
@@ -489,3 +489,72 @@ The total spent voice credit commitment in the MACI contract on-chain is valid
 The per vote option spent voice credit commitment in the MACI contract on-chain is valid
 The total sum of votes in the MACI contract on-chain is valid.
 ```
+
+## Demonstration with `prod-small` settings
+
+Create a MACI instance:
+
+```bash
+node ./build/index.js create -d 0xc87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3 \
+	-sk macisk.8715ab59a3e88a7ceec80f214ec24a95287ef2cb399a329b6964a87f85cf51c \
+	-e http://localhost:8545 \
+    -u 255
+    -m 2048
```

Comment:
It seems that `-u`, `-m` and `-v` parameters are no longer used by the CLI

Reply:
They should be. I'll check.

Reply:
Ah sorry, you're right. 

They're just handled differently now https://github.com/appliedzkp/maci/pull/165/files#diff-6ef4b7073606e835cb72390d47842436L231-L235